src/code/hoc/animate-spring.js

```diff
@@ -16,16 +16,24 @@ import {Motion, spring} from 'react-motion';
     in its target state without further animation. This allows animations at rest to
     respond immediately to window resize events, for instance.
  */
-export default function animateSpring(stiffness=120, damping=25) {
+export default function animateSpring(stiffness=115, damping=18) {
 
   return function(WrappedComponent) {
     return class extends React.Component {
-      
+      componentDidMount() {
+        this.fixedAnimationDuration = window.setInterval(() => {
```

Comment:
Seems like you could use `setTimeout()` since you're only interested in having the timer run once.